### ISSUE TITLE
More pictures of Plants, trees, flowers of Puebla

### ISSUE THREAD

**Issue by philiprbrenan** [Maria-lim04/maria-lim04.github.io#3]
If agreeable, please could you upload more pictures of trees, flowers, plants and animals so that we put them into your web site? This will show the world that Mexico is a nice place.

**Comment by Maria-lim04** [comment]
Yes, of course I would love to share photographs about the nature of Mexico
